docs/index.md

```diff
@@ -64,7 +65,6 @@ trainer_config = TrainerConfig(
     auto_lr_find=True, # Runs the LRFinder to automatically derive a learning rate
     batch_size=1024,
     max_epochs=100,
-    gpus=1, #index of the GPU to use. 0, means CPU
```

Comment:
dropping all these in favor of `accelerator="auto"` which is already in config default :rabbit: